torch/_prims/executor.py

```diff
@@ -49,7 +52,7 @@ def foo(a, b):
     result = traced_foo(a, b, executor='aten')
     """
 
-    def _traced(*args, executor="aten", **kwargs):
+    def _traced(*args: Args, executor: str = "aten", **kwargs: object) -> T:
```

Comment:
```suggestion
    def _traced(*args: P.args, executor: str = "aten", **kwargs: P.kwargs) -> T:
```
where is a ParamSpec object Paramterize by the Callables above